geo/Cargo.toml

```diff
@@ -33,6 +33,8 @@ rstar = "0.12.0"
 serde = { version = "1.0", optional = true, features = ["derive"] }
 i_overlay = { version = "4.0.0, < 4.1.0", default-features = false }
 sif-itree = "0.4.0"
+rand = "0.8.0"
+rand_distr = "0.4.3"
```

Comment:
From what I can tell this is only used in the benchmarks. If so could you move it to `dev-dependencies`?

Reply:
Good catch: we need `rand`, but not `rand_distr`.